<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupSettings">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>400</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="labelOutDir">
         <property name="text">
          <string>Output dir</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelMask">
         <property name="text">
          <string>Input mask</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditMask">
         <property name="placeholderText">
          <string>*.txt, testFile.bin</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelDeleteInputFiles">
         <property name="text">
          <string>Delete input files?</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBoxDeleteInputFiles">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelMode">
         <property name="text">
          <string>Mode</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayoutMode">
         <item>
          <widget class="QRadioButton" name="radioSingle">
           <property name="text">
            <string>Single Run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioTimer">
           <property name="text">
            <string>Timer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelInterval">
         <property name="text">
          <string>Interval (s)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinInterval">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelKey">
         <property name="text">
          <string>XOR Key (hex)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditKey">
         <property name="inputMask">
          <string notr="true">HHHHHHHHHHHHHHHH</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelDuplicate">
         <property name="text">
          <string>On name conflict</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayoutConfilct">
         <item>
          <widget class="QRadioButton" name="radioOverwrite">
           <property name="text">
            <string>Overwrite</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioRename">
           <property name="text">
            <string>Append counter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayoutOutputDir">
         <item>
          <widget class="QLineEdit" name="lineEditOutput">
           <property name="placeholderText">
            <string>ะก:\path\to\output\dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowse">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
